src/common/extensionMessaging.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import * as Q from "q";
+import * as net from "net";
+
+/**
+ * Pipe path used for communicating with the server.
+ */
+let WIN_ServerPipePath = "\\\\?\\pipe\\vscodereactnative";
+let UNIX_ServerPipePath = "/tmp/vscodereactnative.sock";
+
+export let getPipePath = (): string => {
```

Comment:
I'd put this method inside a class. Process can't be mocked with Mockery, so the cleanest option for testing this or dependent methods will probably be to pass a mock process as a constructor argument to some class.